views/components/SelectConfigurationForm.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import Select from '@material-ui/core/Select'
+import MenuItem from '@material-ui/core/MenuItem'
+import FormControl from '@material-ui/core/FormControl'
+
+const SelectConfigurationForm = ({
+  configurations,
+  updatePreferences,
+  currentPreference,
+  classes,
+}) => {
+  return (
+    <form autoComplete='off' className={classes.configForm}>
+      <FormControl className={classes.configFormControl}>
+        <Select
+          value={
+            configurations.filter(
+              (configuration) => configuration._id === currentPreference.config
+            )[0]?._id
+          }
+          onChange={(e) => updatePreferences(e.target.value)}
```

Comment:
should this send the entire configuration object on change?

```suggestion
          onChange={(e) => {
            const configuration = configurations.find(config => config._id === e.target.value)

            updatePreferences(configuration)
          }}
```

Reply:
We don't have to anymore. From your prior suggestion I updated the functions to work with just the configuration._id.

[75fa4e0](https://github.com/AMP-SCZ/dpdash/pull/129/commits/75fa4e051ac01363f4567f566d2f67ddf2280f34)